src/Shared/E2ETesting/E2ETesting.props

```diff
@@ -3,8 +3,16 @@
     <_DefaultProjectFilter>$(MSBuildProjectDirectory)\..\..</_DefaultProjectFilter>
     <DefaultItemExcludes>$(DefaultItemExcludes);node_modules\**</DefaultItemExcludes>
     <SeleniumProcessTrackingFolder Condition="'$(SeleniumProcessTrackingFolder)' == ''">$([MSBuild]::EnsureTrailingSlash('$(RepoRoot)'))artifacts\tmp\selenium\</SeleniumProcessTrackingFolder>
-    <SeleniumE2ETestsSupported Condition="'$(SeleniumE2ETestsSupported)' == '' and '$(TargetArchitecture)' != 'arm' and '$(OS)' == 'Windows_NT' and '$(BuildJava)' == 'true'">true</SeleniumE2ETestsSupported>
-    <EnforcePrerequisites Condition="'$(SeleniumE2ETestsSupported)' == 'true' and '$(EnforcePrerequisites)' == ''">true</EnforcePrerequisites>
+    <SeleniumE2ETestsSupported Condition="'$(SeleniumE2ETestsSupported)' == '' and '$(TargetArchitecture)' != 'arm' and '$(OS)' == 'Windows_NT'">true</SeleniumE2ETestsSupported>
+
+    <!-- We want to enforce prerequisites when we build from the CI or within Visual Studio -->
+    <EnforcedE2EBuildEnvironment Condition="'$(CI)' == 'true' or '$(BuildingInsideVisualStudio)' == 'true'">true</EnforcedE2EBuildEnvironment>
```

Comment:
- `EnforceE2EBuildEnvironment` (no 'd')
- But, why is an additional variable needed? Suggest `EnforceE2ETestPrerequisites` is enough

Reply:
Clarity. It has a d because it is qualifying the environment as a place where E2E test prerequisites should be enforced.

Reply:
Is running E2E tests possible from the command line?

Reply:
Prerequisites will be enforced when calling build.cmd and when running from VS. I'm not sure we can do anything about dotnet test. The tests will still run, even though no prerequisites will be enforced. (Which should display a warning).